fly/fly.py

```diff
@@ -8,17 +9,14 @@
 class Fly:
     def __init__(
             self, concourse_url, executable='/usr/local/bin/fly',
-            platform='darwin', target='default'
+            target='default'
     ):
         self.concourse_url = concourse_url
         self.executable = executable
-        self.platform = platform
+        self.platform = platform.system().lower()
         self.target = target
 
-        if not os.path.isfile(self.executable):
-            self._get_fly()
-
-    def _get_fly(self):
+    def get_fly(self):
         url = f'{self.concourse_url}/api/v1/cli?arch=amd64&platform=' \
             f'{self.platform}'
```

Comment:
I'd replace the `\ ` with parentheses:
```python
url = (
    f'{self.concourse_url}/api/v1/cli?arch=amd64&platform='
    f'{self.platform}'
)
```
actually, as `url` is only used once, maybe just inline it in the `requests.get()` call